app/templates/_bower.json

```diff
@@ -36,24 +36,32 @@
         <%_ if (websocket == 'spring-websocket') { _%>
         "sockjs-client": "1.0.3",
         "stomp-websocket": "2.3.4",
+        <%_ } _%> <%_ if (enableSocialSignIn) { _%>
+        "bootstrap-social": "4.10.1",
         <%_ } _%>
+        "font-awesome": "4.4.0",
```

Comment:
Yes it's good to use Font Awesome, that's what I use for the main JHipster website :-)
But here I don't want to add lots of CSS/images for everyone -> it should be easy to add, but not here by default
@moifort could you just use a String? Or just some specific images?